libcaf_io/caf/io/basp/worker.hpp

```diff
@@ -35,7 +33,7 @@ class CAF_IO_EXPORT worker : public detail::abstract_worker,
 
   using super = detail::abstract_worker;
 
-  using scheduler_type = scheduler::abstract_coordinator;
+  using scheduler_type = scheduler;
```

Comment:
The alias is longer than the actual type. 😉

I'd drop it.

Reply:
Seems it wasn't used.